admin/UpdateDatabasePrivileges.pl

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env perl
+
+use warnings;
+use strict;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+
+use MusicBrainz::Server::Constants qw( @FULL_SCHEMA_LIST );
+use MusicBrainz::Server::Context;
+
+my $c = MusicBrainz::Server::Context->create_script_context(
+    database => 'SYSTEM_MAINTENANCE',
+);
+my $sql = $c->sql;
+
+my $non_select_privileges =
+    'INSERT, UPDATE, DELETE, TRUNCATE, REFERENCES, TRIGGER';
```

Comment:
Something in my brain wants this to be a `qw()` and a `join` but this is probably fine. That said, if you're going to manually `GRANT SELECT` after the `REVOKE`, why not just revoke everything anyway?

Reply:
Ah, ok, didn't realize there's a lot more privileges than these.